server/src/main/java/com/lunchtime/enums/Months.java

```diff
@@ -0,0 +1,16 @@
+package com.lunchtime.enums;
+
+public enum Months {
+    january,
+    february,
+    march,
+    april,
+    may,
+    june,
+    july,
+    august,
+    september,
+    october,
+    november,
+    december
+}
```

Comment:
All enums should be **[uppercase](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html)**.

Reply:
Fix. Change to java.Time.Month enum